services/airflow/dags/datenspende/case_detection_features/extract/extract_survey_data.py

```diff
@@ -0,0 +1,46 @@
+import pandas as pd
+import ramda as R
+
+from database import create_db_context, DBContext
+
+
+@R.curry
+def load_test_and_symptoms_data(questionnaire: int, questions: dict):
+    # select data above from users that have answered the tests and symptoms questionnaire (questionnaire = 10)
+    return R.pipe(
+        lambda *_: create_db_context(),
+        execute_query_and_return_dataframe(
+            """
+                SELECT
+                    answers.user_id, answers.questionnaire_session, answers.question AS question_id,
+                    answers.element as answer_id, questions.text as question, choice.text as answer
+                FROM
+                    datenspende.answers, datenspende.choice, datenspende.questions
+                WHERE
+                    answers.questionnaire != 5 AND
```

Comment:
Because answers about sex, weight, height etc are not coming from questionnaire 10 but from others. questionnaire =! 5 is necessary because in the test data set that we got from thryve, there are some rudiments of a questionnaire 5 in the answers table that does not appear in the questionnaires table.